crates/sui-core/src/authority_aggregator.rs

```diff
@@ -1741,7 +1743,7 @@ where
                                         ?tx_digest,
                                         name=?name.concise(),
                                         expected_epoch=?self.committee.epoch,
-                                        returned_epoch=?inner_signed.auth_sig().epoch,
+                                        returned_epoch=?inner_signed.epoch(),
```

Comment:
whats the difference btw the epoch in auth sig vs the epoch?

Reply:
It's the same thing. Just a shortcut function.